athena-dynamodb/src/main/java/com/amazonaws/athena/connectors/dynamodb/util/DDBTypeUtils.java

```diff
@@ -71,6 +71,8 @@ private DDBTypeUtils() {}
      */
     public static Field inferArrowField(String key, Object value)
     {
+        logger.debug("inferArrowField invoked for key {} with value {} of class {}", key, value,
```

Comment:
is it safe to log values? That might be a security issue

Reply:
removed value